board/crosscompile-toolchain.cmake

```diff
@@ -0,0 +1,41 @@
+## This file handles cross-compilation configurations for aarch64,
+## known as arm64. The objective of this file is to find and assign
+## cross-compiler and the entire toolchain.
+## It works best with buildroot toolchain, when using it the user
+## needs to set the: TOOLCHAIN_PREFIX and CMAKE_SYSROOT from the
+## command line.
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSROOT "" CACHE STRING "CMAKE_SYSROOT")
+set(TOOLCHAIN_PREFIX "" CACHE STRING "TOOLCHAIN_PREFIX")
```

Comment:
Do you think you can clean up the CACHE documentation strings?  They should be, e.g., a short sentence describing what the variable represents.  (A user using the CMake GUI may see these strings, for instance.)  As an example, for this one, I might suggest `Toolchain-specific prefix for compiler and other compiler-related tools`.  (Maybe that could be improved; it's just a suggestion. :))